code/datums/diseases/advance/symptoms/wizarditis.dm

```diff
@@ -0,0 +1,90 @@
+/datum/symptom/wizarditis
+	name = "Wizarditis"
+	desc = "Causes the host to subconciously believe they are in fact, a wizard."
+	stealth = 1
+	resistance = -2
+	stage_speed = -3
+	transmittable = -1
+	level = 0
+	severity = 3
+	symptom_delay_min = 15
+	symptom_delay_max = 30
+	var/teleport = FALSE
+	var/robes = FALSE
+	threshold_desc = "<b>Transmission 14:</b> The host teleports occasionally.<br>\
+					  <b>Speed 7:</b> The host grows a set of wizard robes."
+
+/datum/symptom/wizarditis/Start(datum/disease/advance/A)
+	if(!..())
+		return
+	if(A.properties["transmission"] >= 14)
+		teleport = TRUE
+	if(A.properties["speed"] >= 7)
+		robes = TRUE
+
+/datum/symptom/wizarditis/Activate(datum/disease/advance/A)
+	if(!..())
+		return
+	var/mob/living/M = A.affected_mob
+	switch(A.stage)
+		if(2)
+			if(prob(1) && prob(50))
+				M.say(pick("You shall not pass!", "Expeliarmus!", "By Merlins beard!", "Feel the power of the Dark Side!"))
+			if(prob(1) && prob(50))
+				to_chat(M, "<span class='danger'>You feel [pick("that you don't have enough mana", "that the winds of magic are gone", "an urge to summon familiar")].</span>")
+
+
+		if(3)
+			if(prob(1) && prob(50))
+				M.say(pick("NEC CANTIO!","AULIE OXIN FIERA!", "STI KALY!", "TARCOL MINTI ZHERI!"))
+			if(prob(1) && prob(50))
+				to_chat(M, "<span class='danger'>You feel [pick("the magic bubbling in your veins","that this location gives you a +1 to INT","an urge to summon familiar")].</span>")
+
+		if(4)
+
+			if(prob(1))
+				M.say(pick("NEC CANTIO!","AULIE OXIN FIERA!","STI KALY!","EI NATH!"))
+				return
+			if(prob(1) && robes)
+				to_chat(M, "<span class='danger'>You feel [pick("the tidal wave of raw power building inside","that this location gives you a +2 to INT and +1 to WIS","an urge to teleport")].</span>")
+				spawn_wizard_clothes(50, A)
+			if(prob(1) && prob(50)&&teleport)
+				teleport(A)
+	return
+
+
+
+/datum/symptom/wizarditis/proc/spawn_wizard_clothes(chance = 0, datum/disease/advance/A)
```

Comment:
I kind of think these should have `NODROP` until the disease ends.

Reply:
that is how they are supposed to be, yes